generator/generator_test.go

```diff
@@ -187,7 +193,10 @@ func TestSubControlsMapping(t *testing.T) {
 		},
 	}
 
-	c := (CreateCatalogsFromProfile(&profile))
+	c, err := (CreateCatalogsFromProfile(&profile))
```

Comment:
No need for the extra parenthesis around the function call.

```suggestion
 	c, err := CreateCatalogsFromProfile(&profile)
```

Reply:
Done